### ISSUE TITLE
fix interaction between "tombname" and "untomb"

### ISSUE THREAD

**Issue by stefanhengl** [sourcegraph/zoekt#275]
The shard logs show that some shards are restored just after they have been soft deleted. IE in the logs we see "tombname" followed directly by "untomb". The timestamp show that this happens within the same cleanup run.

```
Fri Feb 4 16:40:13 UTC 2022 0 tombname compound-9b30dd04d89435648576d61010a41d14809b0595_v17.00000.zoekt **repo name**
Fri Feb 4 16:41:00 UTC 2022 0 untomb compound-9b30dd04d89435648576d61010a41d14809b0595_v17.00000.zoekt **repo name**
Sat Feb 5 18:45:59 UTC 2022 0 tombname compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Sat Feb 5 18:47:26 UTC 2022 0 untomb compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Sun Feb 6 18:06:43 UTC 2022 0 tombname compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Sun Feb 6 18:07:27 UTC 2022 0 untomb compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Mon Feb 7 23:23:48 UTC 2022 0 tombname compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Mon Feb 7 23:24:39 UTC 2022 0 untomb compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Tue Feb 8 20:41:18 UTC 2022 0 tombname compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Tue Feb 8 20:42:00 UTC 2022 0 untomb compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Wed Feb 9 10:21:40 UTC 2022 0 tombname compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Wed Feb 9 10:22:19 UTC 2022 0 untomb compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
Wed Feb 9 23:42:00 UTC 2022 0 tombname compound-b69c34db9f4c6dd7a31d6a8105cbfd16bcacc588_v17.00000.zoekt **repo name**
T
```

**Comment by stefanhengl** [comment]
This might have been fixed by #260, which was deployed yesterday. I will keep monitoring the logs to confirm.

**Comment by stefanhengl** [comment]
This is not relevant anymore.
